### ISSUE TITLE
Icon does not show in AlertView. 

### ISSUE THREAD

**Issue by mikeKane** [sparkle-project/Sparkle#1632]
I have my AppIcons setup in my Mac project and when I run 

```
 let updater = SUUpdater.shared()
        updater?.feedURL = URL(string: urlString)
        updater?.checkForUpdates(self)
```

Its presented with a blank white icon. 

Any ideas? 

  - Sparkle (1.23.0)

**Comment by mikeKane** [comment]
The initial check for updates automatically does seem to use the icon, but not the error alert

![Screen Shot 2020-07-28 at 12 21 35 PM](https://user-images.githubusercontent.com/7225808/88693115-ea3d1980-d0cc-11ea-951c-285c366c3a60.png)

**Comment by zorgiepoo** [comment]
Might possibly solved by #1694 in 2.x if it's related to asset catalogs.
